docs/swagger.yaml

```diff
@@ -2022,58 +2145,64 @@ definitions:
   TopRepo:
```

Comment:
please fix all warnings raised by the online editor.  Definition was declared but never used in this document.